bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -409,6 +411,11 @@ private void updateCheckBuilder(CheckBuilder checkBuilder, PullRequestCheckIssue
     }
 
     private boolean updateReadyForReview(PullRequestCheckIssueVisitor visitor, List<String> additionalErrors) {
+        // All the issues must be accessible
+        if (!allIssuesAccessible) {
```

Comment:
I would like to avoid having a boolean field that gets modified at some arbitrary point during a run. We can just send the `regularIssuesMap` to this method and we can check if any of the values is an empty Optional.

Reply:
Sure